evalml/tests/data_checks_tests/test_target_leakage_data_check.py

```diff
@@ -433,8 +438,9 @@ def test_target_leakage_data_check_input_formats_pearson():
     assert leakage_check.validate(X, y.values) == expected
 
 
-def test_target_leakage_none_pearson():
-    leakage_check = TargetLeakageDataCheck(pct_corr_threshold=0.8, method="pearson")
+@pytest.mark.parametrize("measures", ["pearson", "spearman"])
```

Comment:
Should we test across all possible options `["mutual_info", "pearson", "spearman", "max", "all"]`?